### ISSUE TITLE
geojson observations have phenomenonTime start and end, but regular observations do not.  

### ISSUE THREAD

**Issue by jkreft-usgs** [FraunhoferIOSB/FROST-Server#1541]
We are running FROST 2.1.1. Overall, its running much smoother than 1.14.  Thank you for all the improvements!
However, If you look at this geojson call:

https://labs.waterdata.usgs.gov/sta/v1.1/Things?$select=properties,@iot.id&$filter=Datastreams/ObservedProperty/@iot.id%20eq%20%2700060%27%20and%20Locations/properties/mainstemNameAtOutletGNIS%20eq%20756398&$expand=Locations($select=location,@iot.id,properties),Datastreams($select=name,unitOfMeasurement,@iot.id;$filter=ObservedProperty/@iot.id%20eq%20%2700060%27;$top=1;$expand=ObservedProperty($select=name,description,@iot.id),Observations($select=result,parameters,phenomenonTime,@iot.id;$orderby=phenomenonTime%20desc;$top=1))&$resultFormat=GeoJSON&$top=5

and compare it to this regular sensorthings call

https://labs.waterdata.usgs.gov/sta/v1.1/Things?$select=properties,@iot.id&$filter=Datastreams/ObservedProperty/@iot.id%20eq%20%2700060%27%20and%20Locations/properties/mainstemNameAtOutletGNIS%20eq%20756398&$expand=Locations($select=location,@iot.id,properties),Datastreams($select=name,unitOfMeasurement,@iot.id;$filter=ObservedProperty/@iot.id%20eq%20%2700060%27;$top=1;$expand=ObservedProperty($select=name,description,@iot.id),Observations($select=result,parameters,phenomenonTime,@iot.id;$orderby=phenomenonTime%20desc;$top=1))&$top=5

the `phenomenonTime` for each observation in the geojson has a `start` and `end`, which as far as I can tell in the standard should only be at the `Datastreams` metadata, not a per observation thing.    

Here is one feature from the geojson
```
{
      "type": "Feature",
      "properties": {
        "properties/state": "Montana",
        "properties/active": true,
        "properties/agency": "U.S. Geological Survey",
        "properties/county": "Beaverhead County",
        "properties/country": "US",
        "properties/district": "Montana",
        "properties/stateFIPS": "US:30",
        "properties/agencyCode": "USGS",
        "properties/countyFIPS": "US:30:001",
        "properties/countryFIPS": "US",
        "properties/districtCode": "30",
        "properties/altitudeDatum": "North American Vertical Datum of 1988",
        "properties/altitudeMethod": "Interpolated from Digital Elevation Model",
        "properties/hydrologicUnit": "100200010101",
        "properties/altitudeAccuracy": "20",
        "properties/monitoringLocationUrl": "https://waterdata.usgs.gov/monitoring-location/06006000",
        "properties/monitoringLocationName": "Red Rock Cr ab Lakes, nr Lakeview, MT",
        "properties/monitoringLocationType": "Stream",
        "properties/monitoringLocationNumber": "06006000",
        "properties/monitoringLocationAltitudeLandSurface": "6673",
        "@iot.id": "USGS-06006000",
        "Locations/0/@iot.id": "2d86326e-2d27-11ec-8be7-5f0c81fb5a80",
        "Locations/0/properties/type": "hydrolocation",
        "Locations/0/properties/comid": "4172734",
        "Locations/0/properties/measure": 27.08026,
        "Locations/0/properties/comidURL": "https://geoconnex.us/nhdplusv2/comid/4172734",
        "Locations/0/properties/reachcode": "10020001000536",
        "Locations/0/properties/mainstemURL": "https://geoconnex.us/ref/mainstems/312532",
        "Locations/0/properties/mainstemNameAtOutlet": "Missouri River",
        "Locations/0/properties/downstreamMainstemURL": "https://geoconnex.us/ref/mainstems/312091",
        "Locations/0/properties/mainstemNameAtOutletGNIS": 756398,
        "Datastreams/0/name": "7f6427efc9f84cd9a63a2c0bce6a3b4a",
        "Datastreams/0/unitOfMeasurement/definition": "",
        "Datastreams/0/unitOfMeasurement/name": "Cubic Feet per Second",
        "Datastreams/0/unitOfMeasurement/symbol": "ft^3/s",
        "Datastreams/0/@iot.id": "7f6427efc9f84cd9a63a2c0bce6a3b4a",
        "Datastreams/0/ObservedProperty/name": "Discharge, cubic feet per second",
        "Datastreams/0/ObservedProperty/description": "Physical",
        "Datastreams/0/ObservedProperty/@iot.id": "00060",
        "Datastreams/0/Observations/0/parameters/site": "USGS-06006000",
        "Datastreams/0/Observations/0/parameters/masks/0": "SEASONAL",
        "Datastreams/0/Observations/0/phenomenonTime/end": "2023-04-04T20:30:00Z",
        "Datastreams/0/Observations/0/phenomenonTime/start": "2023-04-04T20:30:00Z",
        "Datastreams/0/Observations/0/@iot.id": "7f6427efc9f84cd9a63a2c0bce6a3b4a1680640200"
      }
```

**Comment by hylkevds** [comment]
The phenomenonTime in Observations can be either a time instant or a time interval. There seems to be a bug in the GeoJSON formatter that causes it to always encode as an interval.

**Comment by hylkevds** [comment]
The question is, do we encode instants as `/phenomenonTime/start` with no `/end` or just as `/phenomenonTime`, especially considering instants and intervals may be mixed in a response.

I suspect the former is easier for clients, since they know `/phenomenonTime/start` always exists and can just check for the existence of a `.../end` so see if it's an interval or not.

**Comment by jkreft-usgs** [comment]
I would actually prefer to have the geojson structure reflect the json structure ([and the standard](https://docs.ogc.org/is/18-088/18-088.html#observation)) and have one `phenomenonTime` element that is either a time string or time interval. This also means that there doesn't have to be a bunch of extra documentation around how phenomenonTime handling needs to be special for geojson.


| Name   |      Definition      |  Datatype |
|----------|:-------------:|------:|
| phenomenonTime |  The time instant or period of when the Observation happens. | TM_Object (ISO 8601 Time string or Time Interval string (e.g., 2010-12-23T10:20:00.00-07:00 or 2010-12-23T10:20:00.00-07:00/2010-12-23T12:20:00.00-07:00)) |

**Comment by pbaumard** [comment]
Note that the current [GeoJSON ResultFormat doc](https://fraunhoferiosb.github.io/FROST-Server/extensions/GeoJSON-ResultFormat.html) also uses time interval string in examples:

> ` "Observations/0/phenomenonTime": "2020-03-20T12:00:00.000Z/2020-03-20T13:00:00.000Z",`

**Comment by hylkevds** [comment]
good point, the start/end was a result of the OData work and should not have resulted in changes in the GeoJSON formatter.

**Comment by hylkevds** [comment]
This should be fixed in the released versions 2.1.2 and 2.0.8.
